website/docs/cloud-docs/users-teams-organizations/single-sign-on/index.mdx

```diff
@@ -110,7 +110,7 @@ It is not possible to assign users to the `owners` team through this attribute.
 
 ## Team Names and SSO Team IDs
 
-HCP Terraform expects the team names in the team membership SAML attribute to exactly match its own team names or its configured SSO Team IDs. This match is case sensitive.
+HCP Terraform expects the team names in the team membership SAML attribute to exactly match its own team names or its configured SSO Team IDs. This match is case sensitive and literal, HCP Terraform won't process the value passed by the IdP in any way, therefore values like full DN will not work.
```

Comment:
```suggestion
HCP Terraform expects the team names in the team membership SAML attribute to exactly match team names or configured SSO team IDs stored in HCP Terraform. Values are case sensitive and literal. HCP Terraform does not process the value passed by the IdP. As a result, you cannot use values such as the full distinguished name (DN).
```
Just some style guide related suggestions.

Reply:
Thank you!